src/index.js

```diff
@@ -1,17 +1,13 @@
-import React from 'react';
-import ReactDOM from 'react-dom/client';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import React from "react"
+import ReactDOM from "react-dom"
+//component file
+import TodoContainer from "./components/TodoContainer"
+//stylesheet
+import "./App.css"
 
-const root = ReactDOM.createRoot(document.getElementById('root'));
-root.render(
+ReactDOM.render(
   <React.StrictMode>
-    <App />
-  </React.StrictMode>
-);
-
-// If you want to start measuring performance in your app, pass a function
-// to log results (for example: reportWebVitals(console.log))
-// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
-reportWebVitals();
+    <TodoContainer />
+  </React.StrictMode>,
+  document.getElementById("root")
+)
```

Comment:
[OPTIONAL] Your code seems to use `ReactDOM.render()` to render your components which is no longer supported on newer versions of React (18 and later). This will trigger a warning on the browser and make your application behave like older versions. Now it is is replaced with the `ReactDOM.createRoot()` function and then using `.render()` on the created root. You might also need to import `ReactDOM` from `react-dom/client` instead of `react-dom`.